tutorials/plot_cluster_stats_spatio_temporal_repeated_measures_anova.py

```diff
@@ -198,7 +198,7 @@ def stat_fun(*args):
 
 print('Clustering.')
 T_obs, clusters, cluster_p_values, H0 = clu = \
-    spatio_temporal_cluster_test(X, connectivity=connectivity, n_jobs=1,
+    spatio_temporal_cluster_test(X, connectivity=connectivity, n_jobs=2,
```

Comment:
is it necessary to set n_jobs=2?

we prefer n_jobs=1 in examples as otherwise they will not run out of the box on windows. (not in an "if main" block)